README.md

```diff
@@ -1,126 +1,119 @@
-# replicaservice-typings
-roblox-ts typings for MadStudioRoblox's ReplicaService
+# 🧩 ReplicaService Typings
 
-# Usage
-```ts
-	//Client side, errors if used on server!
-	import { ReplicaController } from "@rbxts/replicaservice"
-
-	//Server side, errors if used on client!
-	import { ReplicaService } from "@rbxts/replicaservice"
-
-	//Somewhere in the project, preferrably a .d.ts file:
-	declare global {
-		interface Replicas {
-			ReplicaName: {
-				Data: {};
-				Tags: {};
-				WriteLib: {};
-			}
-		}
-	}
-	//Modify the Replicas interface to have each Replica class name you use as a key,
-	//and an object with Data, Tags and WriteLib as the value. These objects should then contain the intended data, tags and optional WriteLib
-	//for the replica.
-```
+> `roblox-ts` typings for ReplicaService made by MadStudio.
+
+## 🔢 Table of Contents
+
+- [🔗 TypeScript Example](#-typescript-example)
+- [✨ Recommendations](#-recommendations)
+- [⛔ Limitations](#-limitations)
+- [❔ Frequently Asked Questions](#-frequently-asked-questions)
+- [📜 Documentation](#-documentation)
+- [⚠️ Support](#-support)
```

Comment:
I don't think the emojis are necessary or very helpful, I personally prefer READMEs with less distractions like emojis.